src/widgets/mainwindow.h

```diff
@@ -93,6 +95,7 @@ private slots:
     bool             _useSIPrefixes;
     QTimer           _purgeTimer;
     int              _purgeTimerCount;
+    QMessageBox      *_nukeCountdownWindow;
```

Comment:
How about making it a plain var for now. Connecting its slots in the constructor and in purgeTimerFired just doing .accept()?